x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -445,6 +445,45 @@ class PackagePolicyClientImpl implements PackagePolicyClient {
     };
   }
 
+  public async inspect(
```

Comment:
It would be great to add a comment/rename this function to understand better what it does.

Could you add unit tests as well on the new function?

Reply:
Are you going to add unit tests?